README.md

```diff
@@ -16,29 +16,57 @@ If any visitors have insights into achieving this, please see the issue titled
 
 #Installation
 
-Requirements (in-order):
-* ND4J: https://github.com/deeplearning4j/nd4j
-* ND4S: https://github.com/deeplearning4j/nd4s
-* Scala: v2.10.5+
-* Maven 2.0x+
-* SBT: v0.13.5+
+1. Install Scala: 2.10.x (http://www.scala-lang.org/download/) Be sure to add scala/bin folder to your environment.
+2. Install Spark 1.6.0 which has a Scala 2.10 dependency at http://spark.apache.org/downloads.html.
+3. Add the installation folder to your environment SPARK_HOME = /path/to/installation
+4. Download the latest version of SciSpark from https://github.com/SciSpark/SciSpark
+5. Maven 2.0x+ https://maven.apache.org/download.cgi Be sure to add Maven on your environment.
+6. SBT: v0.13.5+ http://www.scala-sbt.org/download.html
+7. Run ```sbt clean assembly```
+8. Find where your SciSpark.jar (or similarly named) file is and get its path. It is likely under target and then under scala-2.10.
```

Comment:
Hey @valeriearoth, please consider rewording "It is likely under target and then under scala-2.10." to something that indicates the path e.g. /path_to_SciSpark/target/scala-2.10/SciSpark.jar 
To achieve this for e.g. you can make a SciSpark path in your environment to point to the download of SciSpark as a step, then update to $SCISPARK_HOME/target/scala-2.10